content/pt-br/docs/concepts/overview/working-with-objects/cummon-labels.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Recommended Labels
```

Comment:
Please translate "Recommended Labels" to pt-br as well. 😊

![image](https://user-images.githubusercontent.com/46767780/155633470-b519a4a4-a282-42fe-b311-ed732a799077.png)